YYKit/Base/Foundation/NSNumber+YYAdd.m

```diff
@@ -35,9 +35,9 @@ + (NSNumber *)numberWithString:(NSString *)string {
                 @"null" :   [NSNull null],
                 @"<null>" : [NSNull null]};
     });
-    NSNumber *num = dic[str];
+    id num = dic[str];
     if (num) {
-        if (num == (id)[NSNull null]) return nil;
+        if ([num isMemberOfClass:[NSNull class]]) return nil;
```

Comment:
Thanks, get it. "singleton object used to represent null" from http://nshipster.com/nil/